packages/synapse-react-client/src/components/TimelinePlot/TimelinePlot.tsx

```diff
@@ -157,6 +159,9 @@ export const TimelinePlot = ({
     return <NoContentAvailable />
   }
 
+  const widthPx = dimensions.width ? dimensions.width / phaseData.length : 0
+  const gridTemplateColumns = phaseData.map(() => `${widthPx}px`).join(' ')
```

Comment:
Just curious, does this work in a way that `auto auto ...` wouldn't?

Reply:
You're right, I guess I was showing off :) The plot.ly component still needs the actual width, though

Reply:
Updated to `auto auto ... `